programs/solana/programs/ics26-router/src/instructions/client.rs

```diff
@@ -114,7 +125,11 @@ pub fn add_client(
     Ok(())
 }
 
-pub fn update_client(ctx: Context<UpdateClient>, _client_id: String, active: bool) -> Result<()> {
+pub fn migrate_client(
+    ctx: Context<MigrateClient>,
+    _client_id: String,
+    params: MigrateClientParams,
```

Comment:
should client_id be included in params?

Reply:
i separated it so won't need to include client params in account instruction